services/restic_backup.md

```diff
@@ -115,6 +115,34 @@ For mariadb do the following:
 
 ## Restore from a backup
 
+To restore from a GUI 
+
+  1. Install the restic-browser app with 
+
+    ```bash
+    brew install restic-browser
+    ```
+
+  2. Launch the application
+
+     Create a new Location:
+
+       * In the **Type**: Select Google Cloud Storage
+       * In the **Bucket**: Enter the name and path of your bucket. (e.g., gs:postgres-15-backup:daily)
+       * In the **GOOGLE_PROJECT_ID**: Enter pul-gcdc
+       * In the **GOOGLE_APPLICATION_CREDENTIALS**: Get a copy of the credentials and place them at a known location
+         * Log into `lib-postgres-prod1` and download or copy the credentials with the following
+           ```bash
+           sudo su - postgres
+           cat ~/.restic/pul-gcdc-UUIDnumber.json
+           ```
+       * In the **Repository Password**: Get the contents from the server
+         * Log into `lib-postgres-prod1` and get the password with the following
+           ```bash
+           sudo su - postgres
+           cat ~/.restic.pwd
+           ```
+
```

Comment:
Do we check the checkbox 'Insecure TLS (skip TLS certificate verifications) or not?
Do I: 'Save as a new Preset' ?

Reply:
`Make sure the UI has not capitalized any of the input field values; it is case sensitive. Otherwise it will fail to retrieve the repository and it will return an error`

![Restic Browser UI screenshot displaying an error: Failed to open repository](https://github.com/user-attachments/assets/fa673154-f901-4d9f-a0a4-df0a3e661630)

Reply:
After retrieving all the .gz files, how do I download a gz? 
When I  click on 
![Restic-Browser screenshot of retrieved .gz file for lib-jobs](https://github.com/user-attachments/assets/dd699ec4-8c6f-48c9-a4fe-d50d33e012ef)

The following toolbar has a focus:
![Screenshot of Toolbar](https://github.com/user-attachments/assets/887e1d17-1c07-4d64-ae0a-191f4d2d973e)

When I select the one that has the down arrow (title: Dump file/folder contents as zip file), it's prompting me to browse in my directory tree and open a file.